nca/NetworkConfig/ResourcesHandler.py

```diff
@@ -81,8 +82,13 @@ def _set_config_peer_container(self, ns_list, pod_list, resource_list, config_na
             # deepcopy is required since PoliciesFinder may change peer_container
             peer_container = copy.deepcopy(self.global_peer_container)
         else:  # the specific networkConfig has no topology input resources (not private, neither global)
-            print('loading topology objects from k8s live cluster')
-            resources_parser.load_resources_from_k8s_live_cluster([ResourceType.Namespaces, ResourceType.Pods])
+            # this case is reachable when:
+            # 1. no input paths are provided at all, i.e. user intended to get resources from live cluster
+            # 2. paths are provided only using resourceList flag, but no topology objects found;
```

Comment:
"paths are provided only using resourceList flag" - why do we load from live cluster on this case?

Reply:
ok, noticed we do not load on this case